### PR TITLE
Presentation Request improvements

### PR DESCRIPTION
**Pull request by loneil**
https://github.com/bcgov/traction/issues/620#issuecomment-1566510447

Tenants for this PR to test:

Verifier
833b5b90-a958-4d41-bc1e-96f0e63ca105
00ede974-731a-4f08-b7b4-b3a19d6d3e32

Prover
7487318b-f80d-48db-beff-7481003abb97
0dff4464-be8d-42fc-953e-d6f879239153

1. Adjust default pres req template https://github.com/bcgov/traction/issues/616
2. Add a "Copy From existing" button to the rows in the table for pres reqs you created as the verifier https://github.com/bcgov/traction/issues/620
3. Add comment to table and adjust search https://github.com/bcgov/traction/issues/621

image

### CODE REVIEW

File: services/tenant-ui/frontend/src/components/verifier/DeleteExchangeRecord.vue [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  },
+});
+
+// Delete the connection record by connection id
+const deleteRecord = (event: any) => {
+  confirm.require({
+    target: event.currentTarget,
+    message:
+      'Are you sure you want to delete this Presentation Exchange Record?',
+    header: 'Confirmation',
```
Comment (esune) [inline services/tenant-ui/frontend/src/components/verifier/DeleteExchangeRecord.vue:39]: These should be in the `i18n` file

File: services/tenant-ui/frontend/src/components/verifier/DeleteExchangeRecord.vue [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    accept: () => {
+      doDelete();
+    },
+  });
+};
+const doDelete = () => {
+  verifierStore
+    .deleteRecord(props.recordId)
+    .then(() => {
+      toast.success(`Presentation Exchange Record successfully deleted`);
```
Comment (esune) [inline services/tenant-ui/frontend/src/components/verifier/DeleteExchangeRecord.vue:50]: Same as above

**Review by esune — CHANGES_REQUESTED** [review]
If necessary, we can merge this and log an issue to deal with the i18n resources later - changes look great otherwise.

**Comment by swcurran** [comment]
Looks great!  Thanks! I notice that the connection and optional comment did not get copied into the new PR, which is probably fine — just wanted to be sure it was intentional.  Tested it out and it works great. It simplifies my lab instructions for the workshop!

**Review by swcurran — APPROVED** [review]

**Review by esune — APPROVED** [review]

**Pull request merged**